script/test/fixtures/buildargs/docker-compose.yml

```diff
@@ -0,0 +1,17 @@
+version: "2"
+
+services:
+    foo:
+        build: 
+          context: "./build"
+          args:
+            NAME: web
+        command: "sleep 3600"
+
+    foo1:
+            build:
+              context: "./build"
+              args:
+                 - NAME=web
+                 - foo
```

Comment:
@surajnarwade Alright, no worries. Despite it being valid YAML, it's something that's not listed on https://docs.docker.com/compose/compose-file/compose-file-v2/#args :+1: LGTM!